### ISSUE TITLE
Script seems to start fine but get this traceback in terminal after a while

### ISSUE THREAD

**Issue by deronsizemore** [InstaPy/InstaPy#274]
```
Traceback (most recent call last):
  File "quickstart.py", line 21, in <module>
    .like_by_tags(like_tags, amount=25) \
  File "/Desktop/instapy-fs/instapy/instapy.py", line 274, in like_by_tags
    self.username, self.like_by_followers_upper_limit, self.like_by_followers_lower_limit)
  File "/Desktop/instapy-fs/instapy/like_util.py", line 94, in check_link
    post_page = browser.execute_script("return window._sharedData.entry_data.PostPage")
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 487, in execute_script
    'args': converted_args})['value']
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 252, in execute
    self.error_handler.check_response(response)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 194, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 600.000
  (Session info: chrome=58.0.3029.110)
  (Driver info: chromedriver=2.29.461585 (0be2cd95f834e9ee7c46bcc7cf405b483f5ae83b),platform=Mac OS X 10.11.6 x86_64)
```

Here's my code for script:

```
InstaPy(username='', password='') \
    .login() \
    .set_do_follow(True, percentage=60, times=1) \
    .set_lower_follower_count(limit=50) \
    .like_by_tags(like_tags, amount=25) \
    .end()
```

**Comment by sas300** [comment]
I've had this happen recently too. For me it looks like my machine is going into sleep mode and disconnecting the WiFi. Do you still get this problem if you switch off sleep mode (display sleep should be fine)?

**Comment by deronsizemore** [comment]
Thanks. I'm not sure. That was kind of my first thought too. I can't seem to reproduce it now which is weird since it happened a few times over the last week. For example yesterday, I started the script, ended up having to leave so I closed my laptop and put it to sleep and just now opened it back up which also re-connected me to wifi and the script just picked up where it left off yesterday when I stopped. So, I'm not sure. I'll keep an eye on it I guess.
